code/obj/item/mob_parts/robot_parts.dm

```diff
@@ -1216,6 +1212,257 @@ ABSTRACT_TYPE(/obj/item/parts/robot_parts/leg/right)
 /obj/item/parts/robot_parts/robot_frame/syndicate
 	syndicate = TRUE
 
+
+
+/obj/item/parts/robot_parts/drone_frame
+	name = "drone frame"
+	icon_state = "frame"
+	max_health = 5000
+	/// This will make the borg a syndie one
+	var/emagged = 0
+	var/freemodule = TRUE
+	var/build_step = 0
+	var/shelltypetoapply = "eyebot"
+	var/moduletoapply = null
+	var/obj/item/cell/cell = null
+	var/obj/item/ai_interface/ai_interface = null
+	var/obj/item/ = null
```

Comment:
?

Reply:
Huh. Yeah. Question mark, I have no idea what that is, either. Let me see, I can probably get rid of it.